packages/deployer/ens.js

```diff
@@ -46,8 +46,8 @@ class ENS {
       return { resolvedAddress };
     }
     // deploy a resolver if one isn't set
-    const PublicResolverArtifact = require("@ensdomains/resolver")
-      .PublicResolver;
+    const PublicResolverArtifact =
+      require("@ensdomains/resolver").PublicResolver;
```

Comment:
Hm, so we're getting rid of the dependency on `@ensdomains/ens`, but not on `@ensdomains/resolver`?

(I'm actually not clear on *why* we're getting rid of the dependency on `@ensdomains/ens`, given that we seem to just instead be using its contents wholesale... what's the advantage of that?)

Reply:
Eventually we should probably get rid of this as well. These ens packages carry extra baggage given their dependencies. I think the built artifacts in the package are also not current, they were built at a time when a different registry was being used. I don't really think this should make much of a difference, but better to be current anyways I suppose.

Reply:
So what's the difference between the two that results in it making sense to do them one at a time rather than both at once?

Reply:
Oh, huh, so you have a PR to do the other one that targets this one, so I guess ultimately they're both going to be in this one after all?

Reply:
I like PRs that are small, it makes them very easy to review. :)